google/iam.go

```diff
@@ -64,6 +69,36 @@ func iamPolicyReadModifyWrite(updater ResourceIamUpdater, modify iamPolicyModify
 		log.Printf("[DEBUG]: Setting policy for %s to %+v\n", updater.DescribeResource(), p)
 		err = updater.SetResourceIamPolicy(p)
 		if err == nil {
+			fetchBackoff := 5 * time.Second
```

Comment:
I assume you came to this value from trial and error, right? It feels a bit high since that means that at minimum we've just added 15s to this resource, but I'll defer to your findings

Reply:
I can try starting at 1 - increasing the backoff as we run out of quota might work out okay.